tests/populator_test.py

```diff
@@ -120,7 +120,7 @@ def test_get_helper(self, *args):
         """Test get_device_helper for dm devices."""
         device_is_dm = args[0]
         device_is_dm_lvm = args[6]
-        data = Mock()
+        data = {'SYS_PATH': 'dummy'}
```

Comment:
Have you run the unit tests locally to verify this? I would be surprised if this change is sufficient.